app/experimenter/nimbus-ui/src/components/PageResults/TableMetricSecondary/index.tsx

```diff
@@ -132,10 +132,12 @@ const TableMetricSecondary = ({
           })}
         </tbody>
       </table>
-      <GraphsWeekly
-        weeklyResults={results?.weekly!}
-        {...{ probeSetSlug, probeSetName }}
-      />
+      {results?.weekly && (
+        <GraphsWeekly
+          weeklyResults={results?.weekly}
```

Comment:
Not sure if this still needs a "?" in the `results?.weekly` since we're already doing that check above on L135 now. But not sure it hurts anything either, unless TypeScript is terrible at its job.

Reply:
good point, I'll remove it